site/docs/Tutorial-BasicTypes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Basic Types
+layout: ../_core/DocsLayout
+category: Tutorial
+permalink: /graphql-js/basic-types/
+next: /graphql-js/passing-arguments/
+---
+
+In most situations, all you need to specify the types for your API is the GraphQL schema language, taken as an argument to the `buildSchema` function.
+
+The GraphQL schema language supports the scalar types of `String`, `Int`, `Float`, `Boolean`, and `ID`, so you can use these directly in the schema you pass to `buildSchema`.
```

Comment:
Yeah, for using them in actual code you have more stuff to explain though. Like at first I assumed enums mapped to 0, 1, 2 etc because they do in languages like Go and I wondered wasn't that a pain on the client-side. Enums are cool, I just don't think conceptually they are as basic as string / int / bool / float